align_double_arrow looks weird

Since the last update ``align_double_arrow``looks weird

```bash
oskar.stark:/Volumes/development/workspaces/SonataBlockBundle (master)$ make cs
./vendor/bin/php-cs-fixer fix --verbose
Loaded config from "/Volumes/development/workspaces/SonataBlockBundle/.php_cs"
.................FF...........................................................................................
Legend: ?-unknown, I-invalid file syntax, file ignored, .-no changes, F-fixed, E-error
   1) Block/Service/MenuBlockService.php (align_double_arrow)
   2) Block/Service/RssBlockService.php (align_double_arrow)
Fixed all files in 23.432 seconds, 9.500 MB memory used
make: *** [cs] Error 1
oskar.stark:/Volumes/development/workspaces/SonataBlockBundle (master)$ git diff
diff --git a/Block/Service/MenuBlockService.php b/Block/Service/MenuBlockService.php
index 6cd0965..2c88923 100644
--- a/Block/Service/MenuBlockService.php
+++ b/Block/Service/MenuBlockService.php
@@ -133,16 +133,16 @@ class MenuBlockService extends BaseBlockService
     protected function getFormSettingsKeys()
     {
         return array(
-            array('title', 'text', array('required' => false)),
-            array('cache_policy', 'choice', array('choices' => array('public', 'private'))),
-            array('menu_name', 'choice', array('choices' => $this->menus, 'required' => false)),
+            array('title', 'text', array('required'           => false)),
+            array('cache_policy', 'choice', array('choices'   => array('public', 'private'))),
+            array('menu_name', 'choice', array('choices'      => $this->menus, 'required' => false)),
             array('safe_labels', 'checkbox', array('required' => false)),
-            array('current_class', 'text', array('required' => false)),
-            array('first_class', 'text', array('required' => false)),
-            array('last_class', 'text', array('required' => false)),
-            array('menu_class', 'text', array('required' => false)),
-            array('children_class', 'text', array('required' => false)),
-            array('menu_template', 'text', array('required' => false)),
+            array('current_class', 'text', array('required'   => false)),
+            array('first_class', 'text', array('required'     => false)),
+            array('last_class', 'text', array('required'      => false)),
+            array('menu_class', 'text', array('required'      => false)),
+            array('children_class', 'text', array('required'  => false)),
+            array('menu_template', 'text', array('required'   => false)),
         );
     }

diff --git a/Block/Service/RssBlockService.php b/Block/Service/RssBlockService.php
index 80e906e..be76f09 100644
--- a/Block/Service/RssBlockService.php
+++ b/Block/Service/RssBlockService.php
@@ -51,7 +51,7 @@ class RssBlockService extends BaseBlockService
     {
         $formMapper->add('settings', 'sonata_type_immutable_array', array(
             'keys' => array(
-                array('url', 'url', array('required' => false)),
+                array('url', 'url', array('required'    => false)),
                 array('title', 'text', array('required' => false)),
             ),
         ));
oskar.stark:/Volumes/development/workspaces/SonataBlockBundle (master)$ vendor/bin/php-cs-fixer --version
PHP CS Fixer version 1.10.1 by Fabien Potencier
oskar.stark:/Volumes/development/workspaces/SonataBlockBundle (master)$
```

--- a/Symfony/CS/Fixer/Contrib/AlignDoubleArrowFixer.php
+++ b/Symfony/CS/Fixer/Contrib/AlignDoubleArrowFixer.php
@@ -141,9 +141,25 @@ class AlignDoubleArrowFixer extends AbstractAlignFixer
 
             if ($token->equals(',')) {
                 for ($i = $index; $i < $endAt - 1; ++$i) {
-                    if ($tokens->isArray($i + 1) || false !== strpos($tokens[$i - 1]->getContent(), "\n")) {
+                    if (false !== strpos($tokens[$i - 1]->getContent(), "\n")) {
                         break;
                     }
+
+                    if ($tokens->isArray($i + 1)) {
+                        $arrayStartIndex = $tokens[$i + 1]->isGivenKind(T_ARRAY)
+                            ? $tokens->getNextMeaningfulToken($i + 1)
+                            : $i + 1
+                        ;
+                        $blockType = Tokens::detectBlockType($tokens[$arrayStartIndex]);
+                        $arrayEndIndex = $tokens->findBlockEnd($blockType['type'], $arrayStartIndex);
+
+                        $arrayContent = $tokens->generatePartialCode($arrayStartIndex, $arrayEndIndex);
+
+                        if (false !== strpos($arrayContent, "\n")) {
+                            break;
+                        }
+                    }
+
                     ++$index;
                 }
             }

--- a/Symfony/CS/Tests/Fixer/Contrib/AlignDoubleArrowFixerTest.php
+++ b/Symfony/CS/Tests/Fixer/Contrib/AlignDoubleArrowFixerTest.php
@@ -432,6 +432,22 @@ class AlignDoubleArrowFixerTest extends AbstractFixerTestBase
         "utė\b" => "us",
     );',
             ),
+            array(
+                '<?php
+                $formMapper
+                    ->add(\'foo\', null, [\'required\' => false])
+                    ->add(\'dummy_field\', null, [\'required\' => false])
+                ;
+                ',
+            ),
+            array(
+                '<?php
+                $formMapper
+                    ->add(\'foo\', null, array(\'required\' => false))
+                    ->add(\'dummy_field\', null, array(\'required\' => false))
+                ;
+                ',
+            ),
         );
     }
 }

